### ISSUE TITLE
Request for documentation naming available custom checkboxes

### ISSUE THREAD

**Issue by claremacrae** [colineckert/obsidian-things#120]
### tl;dr

If someone could document this theme's supported custom checkboxes and their names -  [something like this](https://github.com/nickmilo/LYT-Mode#alternative-checkboxes) - I would be happy to add support for this theme in the Tasks plugin.

I've been able to figure out the supported checkboxes, but I would want to be able to point Tasks users to a list in the docs here.

### Background

I'm the maintainer of the Tasks plugin, and am adding support for custom statuses, aka custom checkboxes.

The goal is to add more buttons like this, for popular themes with documented custom checkboxes.

![image](https://user-images.githubusercontent.com/4840096/214009367-d9d7bef8-34c7-4aaf-9605-d4cce283ee45.png)

### What I think is supported

I've been looking at the themes whose source code adds custom checkboxes, and have found through experimentation that these change the display:

```text

- [ ] #task ASCII code 32 - `space` 2 - Space
- [!] #task ASCII code 33 - `!` - Exclamation mark
- ["] #task ASCII code 34 - `"` - Double quotes ; Quotation mark ; speech marks
- [*] #task ASCII code 42 - `*` - Asterisk
- [-] #task ASCII code 45 - `-` - Hyphen , minus sign
- [/] #task ASCII code 47 - `/` - Slash , forward slash , fraction bar , division slash
- [<] #task ASCII code 60 - `<` - Less-than sign
- [>] #task ASCII code 62 - `>` - Greater-than sign ; Inequality
- [?] #task ASCII code 63 - `?` - Question mark
- [I] #task ASCII code 73 - `I` - Capital letter I
- [S] #task ASCII code 83 - `S` - Capital letter S
- [X] #task ASCII code 88 - `X` - Capital letter X
- [b] #task ASCII code 98 - `b` - Lowercase letter b , minuscule b
- [c] #task ASCII code 99 - `c` - Lowercase letter c , minuscule c
- [d] #task ASCII code 100 - `d` - Lowercase letter d , minuscule d
- [f] #task ASCII code 102 - `f` - Lowercase letter f , minuscule f
- [i] #task ASCII code 105 - `i` - Lowercase letter i , minuscule i
- [k] #task ASCII code 107 - `k` - Lowercase letter k , minuscule k
- [l] #task ASCII code 108 - `l` - Lowercase letter l , minuscule l
- [p] #task ASCII code 112 - `p` - Lowercase letter p , minuscule p
- [u] #task ASCII code 117 - `u` - Lowercase letter u , minuscule u
- [w] #task ASCII code 119 - `w` - Lowercase letter w , minuscule w
- [x] #task ASCII code 120 - `x` - Lowercase letter x , minuscule x
```

They look like this:

![image](https://user-images.githubusercontent.com/4840096/214009659-1f2faf37-29ac-4925-963d-52fa3c5e34e7.png)

**Comment by colineckert** [comment]
This is a great suggestion, @claremacrae. I just added something like this to the theme `README.md` to showcase what checkbox types are available with Things. Cheers!

**Comment by claremacrae** [comment]
> This is a great suggestion, @claremacrae. I just added something like this to the theme `README.md` to showcase what checkbox types are available with Things. Cheers!

Hi thank you for doing this. I looked at the README and can see an image, but I cannot see text which I canopy and paste. Any chance you could supply the doc that you used to create the screenshot please?

**Comment by colineckert** [comment]
Np, @claremacrae, just added a code block to the readme to copy/paste. Thanks!

**Comment by claremacrae** [comment]
Perfect - thanks very much @colineckert 

For info, your theme will end up with a new page on the Tasks docs equivalent to this Minimal one...
https://obsidian-tasks-group.github.io/obsidian-tasks/reference/status-collections/minimal-theme/

And these are the steps I'll go through to make it look that good - so I am very pleased to be able to copy-and-paste the info...
https://github.com/obsidian-tasks-group/obsidian-tasks/issues/1598#issuecomment-1406589991

**Comment by colineckert** [comment]
Awesome! Thank you @claremacrae!

**Comment by claremacrae** [comment]
I now see from my tests that this is identical to Minimal...

So I am debating between these combinations of buttons:

```
'Add all Status types supported by Minimal Theme'
'Add all Status types supported by Things Theme'
```

Or

```
'Add all Status types supported by Minimal and Things Themes'
```

Then it's obvious to users that they are identical...

But only if they stay identical...

Do you have any thoughts?

**Comment by colineckert** [comment]
this is awesome! ah but that is a dilemma. to save future work, i'd suggest separate buttons. kepano actively develops minimal more than i do things, so there's a chance they won't remain identical.
